Submodels/Legacy/testing_averaging.py

```
@@ -0,0 +1,69 @@
+# Construct output file name
+output_file = f'averaged_file.test_case'
+
+skip_lines: dict = {2, 14, 26, 27, 32, 37, 42}
+
+# Construct a simple local function to load in the contents of each file in the directory
+def read_file(filename):
+    with open(filename, 'r') as file:
+        return file.readlines()
+
+# Read all files in the directory
+file_pattern = f'forces.*'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)